samples/DataSourceWithProcessingAndExtensions/DataExtensionsSample.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29009.5
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataExtensionsSample", "Sample\DataExtensionsSample.csproj", "{F1694A99-71A8-44C2-80F7-7CB1E7EDEAC3}"
```

Comment:
Given that the other two projects are "SampleDriver" and "SampleExtensions", should we rename this to "SampleDataExtensions" for consistency?